lib/IDE/ArgumentCompletion.cpp

```diff
@@ -278,10 +278,21 @@ void ArgumentTypeCheckCompletionCallback::sawSolutionImpl(const Solution &S) {
     }
   }
 
+  bool IncludeSignature = false;
+  if (ParentCall->getArgs()->size() == 1) {
+    Argument Arg = ParentCall->getArgs()->get(0);
+    if (!Arg.hasLabel() && Arg.getExpr() == CompletionExpr) {
```

Comment:
Nit: You can use `getUnlabeledUnaryExpr()` here

Reply:
Oh, nice. I didn’t know about that function.